Asterix/Main_EFC_THD.py

```diff
@@ -339,30 +243,33 @@ def create_interaction_matrices(parameter_file,
         plt.savefig(figSVDEFC)
 
     if onbench == True:
-        # Save EFC control matrix in Labview directory
-        #EFCmatrix = np.zeros((invertGDH.shape[1], DM_pushact.shape[0]),
-        #                     dtype=np.float32)
-        EFCmatrix_DM3 = np.zeros(
-            (invertGDH.shape[1], thd2.DM3.DM_pushact.shape[0]),
-            dtype=np.float32)
+
+        #### Not sure what it does... Is this still useful ?
```

Comment:
Yes, it is! Need to keep as it was! To be discussed.

Reply:
I kept it